test.js

```diff
@@ -1,6 +1,13 @@
 import test from 'ava';
 import invertKeyValue from '.';
 
+test('throws an error if wrong type is given', t => {
+	const error = t.throws(() => {
+		invertKeyValue(null);
+	}, {instanceOf: TypeError});
```

Comment:
You can check the message in the throws assertion.

Reply:
Done (: